torchmetrics/functional/audio/si_sdr.py

```diff
@@ -0,0 +1,60 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import torch
+
+
+def si_sdr(target, estimate, zero_mean=False, EPS=1e-8):
+    """ scale-invariant signal-to-distortion ratio (SI-SDR)
+
+    Args:
+        target (Tensor): shape [..., time]
+        estimate (Tensor): shape [..., time]
```

Comment:
```suggestion
        preds (Tensor): shape [..., time]
```
could we rename `estimate`->`preds` to be consistent with the remaining code base